<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicInformation</class>
 <widget class="QWidget" name="BasicInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="driverButton">
            <property name="text">
             <string>司机信息</string>
            </property>
            <property name="name" stdset="0">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>15</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="truckButton">
            <property name="text">
             <string>货车信息</string>
            </property>
            <property name="name" stdset="0">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="searchBox">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>22</height>
             </size>
            </property>
            <property name="name" stdset="0">
             <string>SelectedButton</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="searchLable">
            <property name="text">
             <string>搜索</string>
            </property>
            <property name="name" stdset="0">
             <string>SearchLable</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit">
            <property name="name" stdset="0">
             <string>SearchLine</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>171</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
